example02/dist.js

```diff
@@ -0,0 +1,134 @@
+function html([first, ...strings], ...values) {
```

Comment:
Please remove this file.

In https://github.com/stasm/innerself/commit/ac3cbf2b49ac1a08be64a8abb7e5f1d35e3ed0d2 I made it such that examples don't need their own `.gitignore` files, so no need to add one here, by the way.